.github/workflows/build.yml

```diff
@@ -146,8 +146,73 @@ jobs:
     - name: Image digest
       run: echo ${{ steps.build.outputs.digest }}
 
+  build-windows:
+    needs: version
+    runs-on: windows-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        base-image:
+          - mcr.microsoft.com/windows/servercore:ltsc2022
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+
+      - name: Set image variables
+        id: image-variables
+        run: |
+          $baseImage = "${{ matrix.base-image }}"
+          $tag = if ($baseImage -like "*nanoserver*") { "nanoserver" } else { "servercore" }
```

Comment:
in your matrix you are not using `nanoserver`. should we include it as well?

Reply:
Had some issues with the nanoserver finding the cache:

https://github.com/jamesfricker/pixi-docker/actions/runs/12128183557/job/33814038072

Not too concerned and happy to proceed without it